biz.aQute.launchpad/test/aQute/launchpad/BundleBuilderTest.java

```diff
@@ -175,4 +175,22 @@ public void testOrder() throws Exception {
 
 		}
 	}
+
+	@Test
+	public void differentBundles_withSameSymbolicName_shouldBeTheSameBundle() throws Exception {
```

Comment:
Shouldn't this method name end in "should _Not_ BeTheSameBundle"?

Reply:
Yes... I thought I had fixed that... oops